Library/Formula/powerline.rb

```diff
@@ -0,0 +1,27 @@
+# Documentation: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md
+#                http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
+
+class Powerline < Formula
+  desc "Powerline is a statusline plugin for vim, and provides statuslines and prompts for several other applications, including zsh, bash, tmux, IPython, Awesome and Qtile."
+  homepage "https://github.com/powerline/powerline"
+  url "https://github.com/powerline/powerline/archive/2.3.tar.gz"
+  version "2.3"
```

Comment:
Homebrew will be able to detect the version here from the URL string, so this isn't necessary.  Running `brew audit --strict --online powerline` will help you catch these things.